### ISSUE TITLE
Comment Errors to be human read 

### ISSUE THREAD

**Issue by KoalaSlippy** [dtube/dtube#211]
As we post or write comment of upvote we see errors like 
could not vote unknow key error 
no one has clue that what that really means or there are few error message we see due to this we dont know the post or anything going trough network this making multiple copies of the post or comments and so on Please make simply 
Ex 
Hello bud we are processing your vote it will be dont shortly
or hey you are commenting to fast i think  you comment will process shortly 

opps seems liek your out vp wait for -- hours or min for refill

something makes us understand ok it will be dont rather then what is this error mean so please work on it

**Comment by d00k13dotcom** [comment]
I would like to broaden the scope of the suggestion, we need transparent alerts for all error messages which have the requirement of clicking confirm to disappear ensuring the user sees them & will slow down button smashing(repeat posting)

**Comment by techcoderx** [comment]
The only solution to repeat posting is to limit creation of new contents to every 5 minutes

**Comment by tibfox** [comment]
Yes thats true but would be a bit restricting for users. I think errors
have to get solved and correctly handled with understandable error messages

techcoderx ***@***.***> schrieb am So., 14. Nov. 2021, 14:03:

**Comment by techcoderx** [comment]
5 minutes per root post is lenient, realistically one would only post a new video at most a few times a day. Comments with parent author and permlink are excluded from this restriction.

**Comment by tibfox** [comment]
With dtube go moments I wouldn't agree on this

techcoderx ***@***.***> schrieb am Mo., 15. Nov. 2021, 06:47:

**Comment by skzap** [comment]
I agree with @tibfox that it's quite annoying to have such a limit. I don't think such a limit should exist.

Most of the duplicate content happens when a node is down, and the user smashes the publish button multiple times because frustrated that it doesn't work... If the author/link is the same, the node will only accept the first transaction and it avoids the issue. The problem is how in some cases it randomizes the <link> (3rd parties I think).

Maybe a simpler solution is to make the <link> deterministic based on the video title or the youtube/facebook id.
